core-web/libs/dotcms-models/src/lib/dot-container.model.ts

```diff
@@ -67,6 +67,6 @@ export interface DotPageContainerStructure {
     [key: string]: {
         container: DotContainer;
         containerStructures?: DotContainerStructure[];
-        contentlets?: { [key: string]: DotCMSContentlet };
+        contentlets?: { [key: string]: DotCMSContentlet[] };
```

Comment:
It was `any` and I encountered errors while mocking this because the mock was an object instead of an array, that's why I found out that it was `any` in some points of the code

Reply:
It's weird because in the places that I used this, since I was using a non object method like `.map()` it was marked as `any` but now that I changed the type to the correct one, it doesn't show `any` anymore. We need to be careful with this `Record-like` types